pages/changelog.md

```diff
@@ -39,7 +39,7 @@ This page is continually updated as changes are made to the BC Data Catalogue an
 
 |**Deployment No.**|**Description**|**Test**|**Production**|
 |:---:|:---|:---:|:---:|
-|18|UI improvements (see [Fixed Issues](https://github.com/bcgov/ckan-ui/blob/master/pages/changelog.md#fixed-issues))|23-March-22||
+|18|UI improvements (see [Fixed Issues](https://github.com/bcgov/ckan-ui/blob/master/pages/changelog.md#fixed-issues))| -April-22||
```

Comment:
Why remove the date when the first PR for release was added and replace it with an empty April date?

Reply:
That was intended for a placeholder for the prod date. i must have entered it wrong.

Reply:
Yeah, prod goes between the next set of bars